tests/files/the_homer.kcd

```diff
@@ -431,5 +431,33 @@
       <Signal name="A" offset="0" length="2" endianess="big"/>
       <Signal name="B" offset="0" length="2" endianess="little"/>
     </Message>
+
+    <Message id="0x900" length="8" name="EnumTestMessage">
+      <Producer>
+        <NodeRef id="1"/>
+      </Producer>
+      <Signal offset="0" length="8" name="EnumTest">
+        <Value slope="2"/>
+        <LabelSet>
+          <Label name="one" value="128"/>
+          <Label name="two" value="255"/>
+        </LabelSet>
+      </Signal>
+   </Message>
+    
+  <Message id="0x901" length="8" name="EnumTestMessage">
+    <Producer>
+      <NodeRef id="1"/>
+    </Producer>
+		
+    <Signal offset="0" length="8" name="EnumTestFloat">
+      <Value slope="2.0"/>
+      <LabelSet>
+        <Label name="one" value="128"/>
+        <Label name="two" value="255"/>
+      </LabelSet>
+    </Signal>
```

Comment:
The Value element should be `<Value type="single" slope="2.0"/>` to make the signal a float, if that is your intention. It's length should be changed to 4 in that case.